membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -112,6 +111,19 @@ class AttributeController(
     )
   }
 
+  private lazy val random = new Random
+
+  private def getZuoraAttributes(identityId: String)(implicit request: AuthenticatedUserAndBackendRequest[AnyContent]) = {
+    if(random.nextFloat > 0.05) {
```

Comment:
do you want it to be random each time or consistent across a given identity id?  Might help if someone is reporting issues as we know if they're "in" our "out"- but would lose the wider coverage.
```scala
    if(List("00", "01", "02", "03", "04").contains(identityId.substring(-2))) {
```

Also I think it will be okay, but in general I've had issues with float/double in java in the past so I default to int where possible
```suggestion
    if(random.nextInt(100) >= 5) {
```